contrib/src/test/java/com/datatorrent/contrib/parser/JsonParserApplicationTest.java

```diff
@@ -0,0 +1,78 @@
+package com.datatorrent.contrib.parser;
```

Comment:
Can you please run mvn apache-rat:check verify -Dlicense.skip=false before uploading the changes?